src/platform/src/Bridge/Scaleway/Scaleway.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Platform\Bridge\Scaleway;
+
+use Symfony\AI\Platform\Capability;
+use Symfony\AI\Platform\Model;
+
+/**
+ * @author Marcus Stöhr <marcus@fischteich.net>
+ */
+final class Scaleway extends Model
+{
+    public const OPENAI_OSS = 'gpt-oss-120b';
+    public const QWEN_INSTRUCT = 'qwen3-235b-a22b-instruct-2507';
+    public const QWEN_CODE = 'qwen3-coder-30b-a3b-instruct';
+    public const GOOGLE = 'gemma-3-27b-it';
+    public const MISTRAL_DEVSTRAL = 'devstral-small-2505';
+    public const MISTRAL_SMALL = 'mistral-small-3.2-24b-instruct-2506';
+    public const MISTRAL_PIXTRAL = 'pixtral-12b-2409';
+    public const META_LLAMA = 'llama-3.3-70b-instruct';
+    public const DEEPSEEK = 'deepseek-r1-distill-llama-70b';
+    public const BAAI_BGE = 'bge-multilingual-gemma2';
+
+    /**
+     * @param array<string, mixed> $options
+     */
+    public function __construct(
+        string $name = self::OPENAI_OSS,
+        array $options = [],
+    ) {
+        $capabilities = [
+            Capability::INPUT_MESSAGES,
+            Capability::INPUT_IMAGE,
+            Capability::INPUT_PDF,
+            Capability::INPUT_MULTIPLE,
+            Capability::OUTPUT_TEXT,
+            Capability::OUTPUT_STREAMING,
+            Capability::OUTPUT_STRUCTURED,
+            Capability::TOOL_CALLING,
```

Comment:
As it's done for other platforms, we will need to provide examples for most of these capabilities. (I'm currently doing it for Perplexity :D )

Reply:
I am in the process of clustering various capabilities to models as not all have the same.